ui/src/containers/BeamlineSetupContainer.jsx

```diff
@@ -151,7 +151,7 @@ function BeamlineSetupContainer(props) {
         <td
           key={`bs-val-${uiprop.attribute}`}
           style={{
-            padding: '0 0.625rem 0 0',
+            padding: '0.125rem 0.625rem 0.125rem 0',
```

Comment:
Adding a little vertical gap between beamline attribute values so the yellow backgrounds (while values are changing) don't merge into one.